libs/coin-modules/coin-stellar/src/api/index.ts

```diff
@@ -97,11 +98,21 @@ async function operationsFromHeight(
     if (state.apiNextCursor) {
       options.cursor = state.apiNextCursor;
     }
-    const [operations, nextCursor] = await listOperations(address, options);
-    const filteredOperations = operations.filter(op => op.tx.block.height >= state.heightLimit);
-    state.accumulator.push(...filteredOperations);
-    state.apiNextCursor = nextCursor;
-    state.continueIterations = operations.length === filteredOperations.length && nextCursor !== "";
+    try {
```

Comment:
Minor : maybe we can scope the try catch clause only around listOp ? (other errors will be thrown anyway, just taste)

Reply:
I had same itch!
Thing is 
```
const [operations, nextCursor] = await listOperations(address, options);
```
We would have to create a `var` before try, then affect the var, then destructure it. Between the 2 poisons I prefered the over-scoped try...